velox/functions/prestosql/tests/RandTest.cpp

```diff
@@ -74,5 +74,140 @@ TEST_F(RandTest, nonNullInt8) {
   EXPECT_LT(rand(4), 4);
 }
 
+class SecureRandTest : public functions::test::FunctionBaseTest {
+ protected:
+  template <typename T>
+  std::optional<T> secureRand(std::optional<T> lower, std::optional<T> upper) {
+    return evaluateOnce<T>("secure_random(c0, c1)", lower, upper);
+  };
+};
+
+TEST_F(SecureRandTest, zeroArgTest) {
```

Comment:
Please don't use Test in the test function names. It can be removed in all.

Reply:
Updated test function names and updated test cases.